examples/example.c

```diff
@@ -0,0 +1,95 @@
+#include <SFML/Audio.h>
+#include <SFML/Graphics.h>
+
+#include <stdlib.h>
+
+int main(void)
+{
+    // Create the main window
+    const sfVideoMode mode = {800, 600, 32};
+    sfRenderWindow* window = sfRenderWindow_create(mode, "SFML window", sfResize | sfClose, NULL);
+    if (!window)
+    {
+        sfRenderWindow_destroy(window);
```

Comment:
There is no window to destroy at this point. We do not need to call `sfRenderWindow_destroy(NULL)`.
Similar remark applies to other error management if statements below.